content/docs/overview.md

```diff
@@ -2489,7 +2489,21 @@ Parametric polymorphism, commonly referred to as "generics", allow the user to c
 ### Explicit parametric polymorphism
 Explicit parametric polymorphism means that the types of the parameters must be explicitly provided.
 #### Procedures
-To specify that a parameter is "constant", the parameters name must be prefixed with a dollar sign `$`. The following example takes two constant parameters to initialize an array of known length:
+As a reminder, all parameters passed into a function are run-time "constants" by default in the sense that they can't have their value changed using `=` directly. Yet, if instead one wishes to use parameter names mutably, then one must indicate so using `:=` like so:
```

Comment:
I would call them immutable rather than run-time constant. Also, "A useful idiom is `var := var`, which expresses a variable shadowing itself. When used at the top of a procedure the compiler understands the use case of enabling local modification of the otherwise immutable variable, and won't complain about the shadowing when you compile with `-vet`."